maven-core/src/main/java/org/apache/maven/rtinfo/internal/DefaultRuntimeInformation.java

```diff
@@ -90,7 +88,9 @@ private String loadMavenVersion() {
 
     @Override
     public boolean isMavenVersion(String versionRange) {
-        Validate.notBlank(versionRange, "versionRange can neither be null, empty nor blank");
+        if (!(versionRange != null && !versionRange.isEmpty())) {
+            throw new IllegalArgumentException("versionRange can neither be null nor empty");
```

Comment:
better to use a NullPointerException for null. The tests below suggest this was the existing behavior.

Reply:
I've always disliked throwing NPE explicitly and this will complicate the code as the test need to be split...  Imho, in all cases, those are illegal arguments...
I'm going to commit the following:
```
        if (Objects.requireNonNull(versionRange, "versionRange cannot be null").isEmpty()) {
            throw new IllegalArgumentException("versionRange cannot be empty");
        }
```

Reply:
That works :-)